docs/src/release-notes-js.md

```diff
@@ -6,6 +6,81 @@ toc_max_heading_level: 2
 
 import LiteYouTube from '@site/src/components/LiteYouTube';
 
+## Version 1.46
+
+### TLS Client Certificates
+
+TLS Client Authentication allows the server to request a client certificate and verify it.
+This release allows you to specify them for the browser and when using the [APIRequestContext] (API Testing).
+The following snippet will add a client certificate for `https://example.com` which gets used in the test runs:
+
+```ts
+import { defineConfig } from '@playwright/test';
+
+export default defineConfig({
+  // ...
+  use: {
+    clientCertificates: [{
+      origin: 'https://example.com',
+      certPath: './cert.pem',
+      keyPath: './key.pem',
+      passphrase: 'mysecretpassword',
+    }],
+  },
+  // ...
+});
+```
+
+Its also possible to manually specify them per-project ([project "use section"](./api/class-testproject#test-project-use)) or when creating a new context or APIRequestContext manually via [`method: Browser.newContext`] and [`method: APIRequest.newContext`].
```

Comment:
```suggestion
It's also possible to specify client certificates per-project ([project "use section"](./api/class-testproject#test-project-use)) or when creating a new context or APIRequestContext manually via [`method: Browser.newContext`] and [`method: APIRequest.newContext`].
```

I don't see what's manual about the per-project `use`, so i've removed that word. Also, replacing `them` with the word it refers to makes this easier to parse when skimming the text rapidly. (general rule for pronouns in technical writing - if it the repetition isn't yet painful, then it's probably a good idea to not use pronous and just repeat)

Reply:
Superseded by Dima's comment.